docs/config.rst

```diff
@@ -435,11 +436,38 @@ The algorithm used by trust manager factory for SSL connections. Default value i
   * Importance: low
 
 Configuration Options for SASL Authentication between REST Proxy and Apache Kafka Brokers
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------------------------------------------------------------------------------
+
+Kafka SASL configurations are described `here <http://docs.confluent.io/3.0.0/kafka/sasl.html>`_
 
 Note that all the SASL configurations (for REST Proxy to Broker communication) are prefixed with "client". If you want the configuration to apply just to consumers or just to producers, you can replace the prefix with "consumer" or "producer" respectively.
 
-In addition to these configurations, make sure ``bootstrap.servers`` configuration is set with SASL_PLAINTEXT://host:port (or SASL_SSL) end-points, or you'll accidentally open an SASL connection to a non-SASL port.
+In addition to these configurations, make sure ``bootstrap.servers`` configuration is set with SASL_PLAINTEXT://host:port (or SASL_SSL://host:port) end-points, or you'll accidentally open an SASL connection to a non-SASL port.
+
+Pass the name of the JAAS file, and the name of Kerberos config file via environment variables to each Kafka broker, Schema Registry, and REST Proxy. For example:
```

Comment:
Can we merge it and  get your help with rewriting to make it clear? Much appreciated!